### PR TITLE
docs: update repository links

### PR DESCRIPTION
**Pull request by 5chdn**

### CODE REVIEW

**Comment by eosclassicteam** [comment]
So, parity moved it's repository twice

First one : https://github.com/ethcore/parity 

Second one : https://github.com/paritytech/parity

Now: https://github.com/paritytech/parity-ethereum

Hope we can see parity-eosclassic someday 😄

**Comment by niklasad1** [comment]
@5chdn 

These would be good to fix too (if they are in scope for this PR):

```bash
.github/CONTRIBUTING.md:Otherwise, just create a [new issue](https://github.com/paritytech/parity/issues/new) in our repository and state:
.github/CONTRIBUTING.md:If you would like to contribute to Parity, please **fork it**, fix bugs or implement features, and [propose a pull request](https://github.com/paritytech/parity/compare).
.github/CONTRIBUTING.md:Please, refer to the [Coding Guide](https://github.com/paritytech/parity/wiki/Coding-guide) in our wiki for more details about hacking on Parity
```
And are we ignoring the `CHANGELOGS`? (it makes sense to me at least)

**Comment by 5chdn** [comment]
> These would be good to fix too (if they are in scope for this PR):

Done.

> And are we ignoring the CHANGELOGS? (it makes sense to me at least)

Yeah it will be redirected anyways.

**Comment by eosclassicteam** [comment]
@niklasad1 @5chdn Please review my PR https://github.com/paritytech/parity-ethereum/pull/9158 👍

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Parity-Ethereum - a fast, light, and robust EVM and WASM blockchain client
 
-### [» Download the latest release «](https://github.com/paritytech/parity/releases/latest)
+### [» Download the latest release «](https://github.com/paritytech/parity-ethereum/releases/latest)
 
 [![build status](https://gitlab.parity.io/parity/parity/badges/master/build.svg)](https://gitlab.parity.io/parity/parity/commits/master)
 [![codecov](https://codecov.io/gh/paritytech/parity/branch/master/graph/badge.svg)](https://codecov.io/gh/paritytech/parity)
```
Comment (ordian) [inline README.md]: should it also be renamed?

**Pull request merged**